test/unit_tests/location_service/set_distance_between_user_and_games_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:niira/models/location.dart';
+import 'package:niira/services/location_service.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:niira/utilities/calc_distance.dart';
+import 'package:test/test.dart';
+
+import '../../mocks/data/mock_games.dart';
+
+void main() {
+  test(
+      'locationService.setDistanceBetweenUserAndGames returns list of games with distanceFromUser set',
+      () {
+    // init mock service
```

Comment:
are these mocks? they seem to be real objects?

Reply:
the next two lines below I mean

Reply:
very real they are haha good pick up!